x/params/store/store.go

```diff
@@ -0,0 +1,177 @@
+package store
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Individual parameter store for each keeper
```

Comment:
Can we describe why we need a transient key and a standard key, and what each is for?